Sources/_AtomicsShims/src/_AtomicsShims.c

```diff
@@ -16,6 +16,14 @@
 // like calls to swift_retain_n/swift_release_n appearing in Swift code, not
 // even when imported through C. (See https://bugs.swift.org/browse/SR-13708)
 
+#if defined(__APPLE__) && defined(__MACH__)
+// Ensure we link with libswiftCore.dylib even when the build system decides
+// to build this module as a standalone library.
+// (See https://github.com/apple/swift-atomics/issues/55)
+__asm__(".linker_option \"-L/usr/lib/swift\"\n");
```

Comment:
Ah, `-L` doesn't actually work, but (for some reason) it also seems unnecessary, at least on 5.9.